app/Logger.ts

```diff
@@ -0,0 +1,70 @@
+import winston from 'winston';
+import * as Transport from 'winston-transport';
+import expressWinston from 'express-winston';
+import {Handler} from "express";
+import chalk from 'chalk';
+
+export default class Logger {
+
+    private readonly logger: winston.Logger;
+    private readonly transport: Transport = new winston.transports.Console();
+    private readonly format = winston.format.combine(
+        winston.format.timestamp({
+            format: 'DD.MM.YYYY - HH:mm:ss'
```

Comment:
Preferably we should use the international standard here, which would reverse the date part.